### PR TITLE
Bootstrap account admin

### PR DESCRIPTION
**Pull request by rahearn**
**Description of change**

Adds a script for bootstrapping the first account admin. This script hardcodes the admin's email, so that calling it with a different admin would require a full PR review and deployment.

**How to test**

* Run the backend tests

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/243

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [x] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

**Comment by kryswisnaskas** [comment]
Nice!
There are a few places where `eslint `complains, but functionally, it works.

File: tools/bootstrapAdmin.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  const user = await User.findOne({ where: { email: ADMIN_EMAIL } });
+  if (user === null) {
+    throw new Error(`User ${ADMIN_EMAIL} could not be found to bootstrap admin`);
+  }
+
+  const [permission, created] = await Permission.findOrCreate({
+    where: {
+      userId: user.id,
+      scopeId: ADMIN,
+      regionId: 14
```
Comment (kryswisnaskas) [inline tools/bootstrapAdmin.js:19]: 👍 We should be ok if this is limited to region 14.

**Review by kryswisnaskas — APPROVED** [review]

File: tools/bootstrapAdmin.js (hunk 1 above)
Comment (rahearn) [reply tools/bootstrapAdmin.js:19]: I thought `14` was meant for "non-regional" things. Should this actually be something else?

**Comment by rahearn** [comment]
> There are a few places where `eslint `complains

Hmm, where do you see that? https://app.circleci.com/pipelines/github/HHS/Head-Start-TTADP/155/workflows/d340a7b1-edb6-4dfc-ab12-72f704e6be2b/jobs/856 looks clean, so if there are linter complaints we should probably bring the test step into alignment with what you're seeing.

File: tools/bootstrapAdmin.js (hunk 1 above)
Comment (jasalisbury) [reply tools/bootstrapAdmin.js:19]: Yeah `14` is for "non-regional" permissions

Comment (jasalisbury) [reply tools/bootstrapAdmin.js:19]: > Yeah `14` is for "non-regional" permissions

Which doesn't feel that great... maybe we should allow the region to be null?

Comment (rahearn) [reply tools/bootstrapAdmin.js:19]: > Which doesn't feel that great (wish I had come up with something better!)... maybe we should allow the region to be null?

I think this is something to revisit as we work on more central office-focused stories.

File: package.json [hunk 2]
```diff
@@ -86,7 +87,7 @@
     ],
     "coverageThreshold": {
       "global": {
-        "branches": 70
+        "branches": 75
```
Comment (jasalisbury) [inline package.json:90]: :chart_with_upwards_trend:

**Review by jasalisbury — APPROVED** [review]

**Pull request merged**